bad/source/lex.rs

```diff
@@ -12,12 +12,14 @@ pub enum TokenName {
 	Identifier,
 	#[regex(r"[0-9]+")]
 	Number,
-	#[regex(r"'[^']+'")]
+	#[regex(r"'[^'\\]*(?:\\.[^'\\]*)*'")]
 	StringLiteral,
 	#[token("/*")]
 	CommentStart,
 	#[token("*/")]
 	CommentEnd,
+	//#[regex(r"/\*.*\*/")]

```

Comment:
Should this be a lazy matcher, e.g. `/\*.*?\*/`? I think you want to stop at the first occurence of `*/`.